dockstore-webservice/src/main/java/io/dockstore/webservice/resources/TokenResource.java

```diff
@@ -401,6 +408,17 @@ public Token addGoogleToken(@ApiParam(hidden = true) @Auth Optional<User> authUs
         return dockstoreToken;
     }
 
+    private void acceptTOSAndPrivacyPolicy(User user) {
+        if (user.getTOSVersion() != CURRENT_TOS_VERSION) {
+            user.setTOSVersion(CURRENT_TOS_VERSION);
+            user.setTOSVersionAcceptanceDate(new Date());
```

Comment:
Create date once and use it twice. That way the acceptance time will be the same for both.